source/locales/es/LC_MESSAGES/carbonstorage.po

```diff
@@ -0,0 +1,1232 @@
+"the year of the current landscape and \"T\" the year of the future "
+"landscape. With positive discounting, the value of the modeled path (the "
+"solid line) is less valuable than the actual path. Therefore, if "
+"sequestration paths tend to follow the dotted line, the model will "
+"undervalue sequestered carbon.*"
+msgstr ""
+
+#: ../../source/carbonstorage.rst:72
+msgid "Data Needs"
+msgstr ""
+
+#: ../../source/carbonstorage.rst:74
+msgid ""
+"**Current land use/land cover** (required): Raster of LULC for each "
+"pixel, where each unique integer represents a different class. *All "
+"values in this raster MUST have corresponding entries in the Carbon Pools"
+" table.*"
+msgstr ""
+
+#: ../../source/carbonstorage.rst:76
+msgid ""
+"**Current Landcover Calendar Year** (required for sequestration): The "
+"year depicted by the Current LULC map, for use in economic valuation."
+msgstr ""
+
+#: ../../source/carbonstorage.rst:78
+msgid ""
+"**Carbon pools** (required): A CSV (comma-separated values) table of LULC"
+" classes, containing data on carbon stored in each of the four "
+"fundamental pools for each LULC class. If information on some carbon "
+"pools is not available, pools can be estimated from other pools, or "
+"omitted by leaving all values for the pool equal to 0. The table must "
+"contain the following columns:"
+msgstr ""
+
+#: ../../source/carbonstorage.rst:80
+msgid ""
+"**lucode**: Unique integer for each LULC class (e.g., 1 for forest, 3 for"
+" grassland, etc.) *Every value in the LULC map MUST have a corresponding "
+"\"lucode\" value in the Carbon Pool table.*"
+msgstr ""
+
+#: ../../source/carbonstorage.rst:81
+msgid ""
+"**c_above**: Carbon density in aboveground biomass [units: "
+"megagrams/hectare]"
+msgstr ""
+
+#: ../../source/carbonstorage.rst:82
+msgid ""
+"**c_below**: Carbon density in belowground biomass [units: "
+"megagrams/hectare]"
+msgstr ""
+
+#: ../../source/carbonstorage.rst:83
+msgid "**c_soil**: Carbon density in soil [units: megagrams/hectare]"
+msgstr ""
+
+#: ../../source/carbonstorage.rst:84
+msgid "**c_dead**: Carbon density in dead matter [units: megagrams/hectare]"
+msgstr ""
+
+#: ../../source/carbonstorage.rst:86
+msgid ""
+"*Example:* Hypothetical study with five LULC classes. Class 1 (Forest) "
+"contains the most carbon in all pools. In this example, carbon stored in "
+"above- and below-ground biomass differs strongly among land use classes, "
+"but carbon stored in soil varies less dramatically."
+msgstr ""
+
+#: ../../source/carbonstorage.rst:89
+msgid "lucode"
+msgstr ""
+
+#: ../../source/carbonstorage.rst:89
+msgid "LULC_name"
+msgstr ""
+
+#: ../../source/carbonstorage.rst:89
+msgid "C_above"
+msgstr ""
+
+#: ../../source/carbonstorage.rst:89
+msgid "C_below"
+msgstr ""
+
+#: ../../source/carbonstorage.rst:89
+msgid "C_soil"
+msgstr ""
+
+#: ../../source/carbonstorage.rst:89
+msgid "C_dead"
+msgstr ""
+
+#: ../../source/carbonstorage.rst:91
+msgid "1"
+msgstr ""
+
```

Comment:
Interesting to see how tables get broken out.

Reply:
Maybe I should go through and remove the tables from the PO files... `c_dead` for instance should not be translated.

Reply:
I wish `sphinx-intl` had a configuration to allow for ignoring certain RST syntax, like tables. Removing table sections from the PO files each time they need to be updated seems fairly onerous. A quick google search didn't bring up anything related unfortunately.

Reply:
I wonder if it would be easier not to use `gettext` at all for the RST docs and try to get them translated directly by someone who's familiar with RST syntax.

Reply:
I made a ticket to revisit this in the future: #54